tests/conftest.py

```diff
@@ -3,17 +3,53 @@
 import getpass
 import random
 
+from multiprocessing.managers import SyncManager
+from loky import get_reusable_executor
+
 from .helpers import gsutil_path_exists, gsutil_wipe_path
 
 import bionic as bn
+from bionic.decorators import persist
+
+
+@pytest.fixture(scope="session")
+def executor():
+    return get_reusable_executor()
+
+
+@pytest.fixture(scope="session")
+def manager():
+    class MyManager(SyncManager):
+        pass
+
+    # MyManager.register('Counter', ResettingCounter)
+    # MyManager.register('TaskKeyLogger', TaskKeyLogger)
+    manager = MyManager()
+    manager.start()
+    return manager
 
 
 # We provide this at the top level because we want everyone using FlowBuilder
 # to use a temporary directory rather than the default one.
 @pytest.fixture(scope="function")
-def builder(tmp_path):
+def builder(executor, manager, tmp_path):
     builder = bn.FlowBuilder("test")
     builder.set("core__persistent_cache__flow_dir", str(tmp_path / "BNTESTDATA"))
+
+    # NOTE: This is disabled till we are ready to switch to parallel processing for tests.
```

Comment:
I was thinking more like we will run all the tests with and without executor / manager using [parameterized](https://docs.pytest.org/en/latest/parametrize.html#pytest-generate-tests) builders. This would be the builder that takes the manager and there will be another one without manager.

Reply:
I agree, that makes sense. (Although we might still want the parameterization to be configurable, since this will significantly increase the running time of the tests.)

In the shorter term, though -- assuming you don't already have a PR ready to do the above -- it would be nice to be able to have an interface for this other than commenting/uncommenting code. Otherwise we'll have to keep remembering to uncomment the code, make changes, then re-comment before committing.